tests/Nox.Integration.Tests/DatabaseIntegrationTests/SqlServerIntegrationTests.cs

```diff
@@ -8,10 +8,11 @@
 using Guid = Nox.Types.Guid;
 
 namespace Nox.Integration.Tests.DatabaseIntegrationTests;
-
-public class SqlServerIntegrationTests : SqlServerTestBase
+[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1000:Test classes must be public", Justification = "Disabled")]//*/
+/*
+public /**/ class SqlServerIntegrationTests : SqlServerTestBase
 {
-    //[Fact]
+    [Fact]
```

Comment:
you need to comment or skip before integration, because ci pipeline is not able to run the tests for now

Reply:
I converted this class to private so that its tests will be ignored.  I think it is easier to manage it with this approach. What do you think ?

[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1000:Test classes must be public", Justification = "Disabled")]//*/
/*
public /**/  class  PostgresIntegrationTests : PostgresTestBase